app/views/work_orders/_send_to.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="col-xs-12">
+  <label>Send to:</label><br />
+</div>
+<div class="col-xs-12">
+  <div class="row">
+    <% if @job.job_coordinator %>
+      <div class="col-sm-3">
+        <div class="form-group">
+          <%= select_tag 'work_order_send_to[send_to][]', options_for_select(User.where(id: @job.job_coordinator.id).collect{ |u| [u.full_name, u.id] }), class: "form-control" %>
+        </div>
+      </div>
+    <% end %>
+    <div class="col-sm-3">
+      <div class="form-group">
+        <%= select_tag 'work_order_send_to[dont_send_to][]', options_for_select(User.where(email: "kroggemann@servpro5933.com").collect{ |u| [u.full_name, u.id] }), class: "form-control" %>
+      </div>
+    </div>
+    <div class="col-sm-3">
+      <div class="form-group">
+        <%= select_tag 'work_order_send_to[send_to][]', options_for_select(User.with_role('Crew Chief').collect{ |u| [u.full_name, u.id] }), prompt: "Select Crew Chief", class: "form-control" %>
+      </div>
+    </div>
+    <div class="col-sm-3">
+      <div class="form-group">
+        <%= select_tag 'work_order[vendor_id]', options_for_select(Vendor.all.collect{ |u| [u.name, u.id] }), prompt: "Select Vendor", class: "form-control" %>
```

Comment:
Definitely agree here, this is another bit that was just moved for the time being - as I mentioned I didn't want to take the refactoring too far initially, so some bits like this for example have just been moved instead of rewritten.

Thanks for the comments by the way though, I appreciate it!